generic/tests/ioquit.py

```diff
@@ -2,10 +2,14 @@
 import time
 import random
 
-from autotest.client.shared import error
+from virttest import qemu_storage
+from virttest import data_dir
+from virttest import error_context
+from avocado.utils import process
+from avocado.core import exceptions
 
 
-@error.context_aware
+@error_context.context_aware
 def run(test, params, env):
     """
     Emulate the poweroff under IO workload(dd so far) with signal SIGKILL.
```

Comment:
Please update the doc as you added image check in your test.

Reply:
@pingl2016, hi, pingl, i have updated the doc and test.fail msg, please help to review again, thank you.